src/ferui-components/tree-view/tree-view-component.ts

```diff
@@ -466,4 +475,12 @@ export class FuiTreeViewComponent<T> implements OnInit, OnDestroy {
       }
     });
   }
+
+  /**
+   * Calls the Tree View auto selector callback for developer to choose an initial data node to select
+   * @param nodeArray
+   */
+  private callAutoSelector(nodeArray: Array<TreeNodeData<T>>): void {
```

Comment:
Why do you need to create a function for that ? 
You can just call `this.selectNode(this.autoSelector.autoSelectNode(nodeArray));` right away instead.

Reply:
Fixed